_posts/2020-10-07-torch-modules/torch_modules.Rmd

```diff
@@ -0,0 +1,429 @@
+---
+title: "Using torch modules"
+description: > 
+ In this next-to-last installment of a mini-series introducing torch basics, we replace hand-coded matrix operations by modules, considerably simplifying our toy network's code. 
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydanatorchmodules
+date: 10-07-2020
+categories:
+  - Torch
+  - R
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+preview: images/preview.jpg
+editor_options: 
+  markdown: 
+    wrap: 72
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+[Last
+week](https://blogs.rstudio.com/ai/posts/2020-10-01-torch-network-from-scratch),
+we started learning about `torch` basics by coding a simple neural
+network from scratch, making use of just a single of `torch`'s features:
+*tensors*. [On
+Monday](https://blogs.rstudio.com/ai/posts/2020-10-05-torch-network-with-autograd),
+we immensely simplified the task, replacing manual backpropagation with
+*autograd*. Today, we *modularize* the network - in both the habitual
+and a very literal sense: Low-level matrix operations are swapped out
+for `torch` `module`s.
```

Comment:
Since this post will (presumably) be read in the future, I think you're better off referencing previous posts as first and second (or something of the like)—last week and Monday aren't very meaningful without context.

Reply:
makes sense!